drastic-price-change-template/README.md

```diff
@@ -0,0 +1,25 @@
+# Drastic Price Change Anomaly Detection Bot
+
+## Description
+
+This bot detects if the price of an asset changes drastically or there is a large discrepancy between an on-chain and an off-chain oracle 
+
+## Supported Chains
+
+- Ethereum (For now)
+
+## Alerts
+
+- PRICE-DISCREPANCIES
+  - Fired when there is a large discrepancy between an on-chain and an off-chain oracle
+  - Severity is always set to "high"
+  - Type is always set to "suspicious"
+
+- PRICE-FLUCTUATIONS
+  - Fired when the price of an asset changes drastically
+  - Severity is always set to "low"
+  - Type is always set to "suspicious"
+
+## Test Data
+
+The bot behaviour can be verified with the provided unit tests
```

Comment:
i think inverse finance would be a good one to validate this on

Reply:
as well as LUNA token (chainlink circuit broke their oracle as soon as the price dropped below 0.1 whereas on centralized exchanges it was well below that)

Reply:
Chainlink no longer provide price feed for LUNA, I used the wayback machine to get the old contract, but it is LUNA/ETH and we would have to modify the bot a lot to work with ETH oracle so I don't think it is worth testing